operator/internal/controller/vectorized/cluster_controller_attached_resources.go

```diff
@@ -126,15 +126,6 @@ func (a *attachedResources) getBootstrapServiceKey() types.NamespacedName {
 	return a.getBootstrapService().Key()
 }
 
-func (a *attachedResources) clusterRole() {
```

Comment:
Please also remove the underlying resource implementations now that they've been replaced. The majority of this should be fairly clean to simply rip out. There's one that's a bit more difficult (Bootstrap LB IIRC) that we can leave in place for simplicity _but_ it's `obj` implementation should delegate to your newly added method instead of having two separate implementations.